extension/src/plots/model/index.ts

```diff
@@ -105,8 +102,8 @@ export class PlotsModel extends ModelWithPersistence {
     this.customPlotsOrder = this.revive(PersistenceKey.PLOTS_CUSTOM_ORDER, [])
   }
 
-  public transformAndSetExperiments(data: ExperimentsOutput) {
-    this.recreateCustomPlots(data)
+  public transformAndSetExperiments() {
```

Comment:
[I] As discussed on the call this morning.

I think this can now be dropped and `recreateCustomPlots` can be renamed/called instead of `getCustomPlots`. It will get called from `sendWebviewMessage` and it can be passed the selected revisions so that we only need to process the required experiments. Instead of processing them all and then filtering them afterwards.

Reply:
Since this pr already has approval and all other comments have been addressed, I completed this change in followup pr, #3491